command/meta.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strings"
 
 	"github.com/hashicorp/nomad/api"
+	isatty "github.com/mattn/go-isatty"
```

Comment:
Would this golang package work similarly? https://godoc.org/golang.org/x/crypto/ssh/terminal#IsTerminal